pallets/protos/Cargo.toml

```diff
@@ -100,6 +100,14 @@ serde_json = { version = '1.0.79', default-features = false, features = ['alloc'
 protos = { version = "0.1.11", default-features = false }
 base58 = { version = "0.2.0", default-features = false }
 
+# [dev-dependencies]
+# sp-clamor = { version = '0.1.0', path = '../../primitives/clamor'}
+# pallet-detach = {path = '../detach'}
+# pallet-frag = {path = '../frag'}
+
+[dev-dependencies]
+pallet-tickets = {path = '../tickets', default-features = false, features=['compile-dummy-data']}
```

Comment:
this is already a normal dependency no? should be.

Reply:
Yes. But the normal dependency doesn't enable the feature "compile-dummy-data"

(Dev-Dependencies only run when doing `cargo test` btw)